docs/operator-manual/applicationset/Appset-Any-Namespace.md

```diff
@@ -0,0 +1,180 @@
+# ApplicationSet in any namespace
+
+**Current feature state**: Beta
+
+!!! warning
+    Please read this documentation carefully before you enable this feature. Misconfiguration could lead to potential security issues.
+
+## Introduction
+
+As of version 2.8, Argo CD supports managing `ApplicationSet` resources in namespaces other than the control plane's namespace (which is usually `argocd`), but this feature has to be explicitly enabled and configured appropriately.
+
+Argo CD administrators can define a certain set of namespaces where `ApplicationSet` resources may be created, updated and reconciled in. 
+
+As Applications generated by an ApplicationSet are generated in the same namespace as the ApplicationSet itself, this works in combination with [App in any namespace](../app-any-namespace.md).
+    
+## Prerequisites
+
+### App in any namespace configured
+
+This feature needs [App in any namespace](../app-any-namespace.md) feature activated. The list of namespaces must be the same.
+
+### Cluster-scoped Argo CD installation
+
+This feature can only be enabled and used when your Argo CD ApplicationSet controller is installed as a cluster-wide instance, so it has permissions to list and manipulate resources on a cluster scope. It will *not* work with an Argo CD installed in namespace-scoped mode.
+
+## Implementation details
+
+### Overview
+
+In order for an ApplicationSet to be managed and reconciled outside the Argo CD's control plane namespace, two prerequisites must match:
+
+1. The namespace list from which `argocd-applicationset-controller` can source `ApplicationSets` must be explicitly set using environment variable `ARGOCD_APPLICATIONSET_NAMESPACES` or alternatively using parameter `--applicationset-namespaces`.
+2. The enabled namespaces must be entirely covered by the [App in any namespace](../app-any-namespace.md), otherwise the generated Applications generated outside the allowed Application namespaces won't be reconciled
+
+It can be achieved by setting the environment variable `ARGOCD_APPLICATIONSET_NAMESPACES` to argocd-cmd-params-cm `application.namespaces`
```

Comment:
I wonder if we should have a separate `applicationset.namespaces` field, in case someone (for some reason) wants those namespaces to be distinct. I could imagine someone wanting applicationset to touch only a subset of all namespaces.

Reply:
That's a really good suggestion. I will do it.

Reply:
```suggestion
It can be achieved by setting the environment variable `ARGOCD_APPLICATIONSET_NAMESPACES` to argocd-cmd-params-cm `applicationset.namespaces`
```

I think we can make this change now?

Reply:
It should also be added to docs/operator-manual/argocd-cmd-params-cm.yaml.